identity_base_config/files/default/id-rails-console

```diff
@@ -40,6 +44,24 @@ if [ $# -ge 1 ]; then
     esac
 fi
 
+while true; do
+    read -r -p "Please provide an explanation for this Rails console: " USE_CASE
+    if [ ! -z "$USE_CASE" ]; then
+        break
+    fi
+done
+
+CHANNEL=$(cat /etc/login.gov/keys/slackchannel) || ((missing_keys++))
+WEBHOOK=$(cat /etc/login.gov/keys/slackwebhook) || ((missing_keys++))
+if [ ${missing_keys} -gt 0 ]; then
+    echo 'Slack channel/webhook missing! Verify files and run Chef to repopulate.'
+    exit 1
+fi
+
+TEXT=":rails:*rails console*:rails: for \`$(logname)\` on \`$(hostname)\`\nuse case: \\\"${USE_CASE}\\\""
```

Comment:
Do we need to worry about escaping characters in USE_CASE? What will happen if someone includes typical punctuation or newlines?

Reply:
I added (and tested) a line to parse and escape characters using `printf '%q'` and reassign that value to `$USE_CASE`:
```
:~$ USE_CASE="Testing this
> weird thing! What 'will' it do?"
:~$ printf '%q\n' "$USE_CASE"
$'Testing this\nweird thing! What \'will\' it do?'
:~$ USE_CASE=$(printf '%q\n' "${USE_CASE}" | sed 's/\$//')
:~$ echo $USE_CASE
'Testing this\nweird thing! What \'will\' it do?'
```